app/assets/stylesheets/application.css.scss

```diff
@@ -35,4 +35,17 @@
 
 .block_user_current_block_id {
   margin-left: 20px;
-}
\ No newline at end of file
+}
+
```

Comment:
Avoid using id selectors<br>Selector `flickr_image` should be written in lowercase with hyphens